src/app/lib/config/toggles/liveConfig.js

```diff
@@ -72,4 +72,7 @@ export default {
   webVitalsMonitoring: {
     enabled: true,
   },
+  topBarOJs: {
```

Comment:
Can we put the toggle in alphabetic order in this (and other) config files?

Reply:
Done